test/powershell/Modules/Microsoft.PowerShell.Utility/Measure-Object.Tests.ps1

```diff
@@ -64,7 +64,7 @@ Describe "Measure-Object" -Tags "CI" {
     }
 
     It "Should be able to count using the Property switch" {
-        $expected = $(Get-ChildItem $TestDrive).Length
+        $expected = $(Get-ChildItem $TestDrive | Where-Object { $_ -is [System.IO.FileInfo] }).Length
```

Comment:
I'd say it is a bug in Measure-Object cmdlet because DirectoryInfo has Length property with value 0.

Reply:
It seems my bad. DirectoryInfo hasn't Length property. FileInfo has.